### PR TITLE
Second round of customization

### PR DESCRIPTION
**Pull request by ThibodeauJF**
**What is missing from Atomic (will be fixed in upcoming days in the source code):**
1. Overriding image size on a specific result template (Academy should have small images) will do a new PR here once it's done
2. Removing the extra space at the bottom of results (see image)
3. Removing the fixed 1/1 aspect ratio for images (see image)
4. Support field conditions for isRecommendation and isTopResult

**What is missing from UX:**
1. List of different badge colors
2. Default Academy placeholder image
3. New Logo

If we ever want to be able to switch the whole interface from English/French, additional work would be required, let me know!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,13 +10,13 @@
 
     <link
       rel="stylesheet"
-      href="https://static.cloud.coveo.com/atomic/v1/themes/coveo.css"
+      href="https://staticdev.cloud.coveo.com/atomic/v1/themes/coveo.css"
```
Comment (ThibodeauJF) [inline index.html:13]: dev will allow us to get new content as soon as it's merged (and when the cache clears ~1 day)

File: index.html [hunk 2]
```diff
@@ -30,6 +30,12 @@
         });
 
         searchInterface.executeFirstSearch();
+
+        searchInterface.i18n.addResourceBundle("en", "caption-source", {
```
Comment (ThibodeauJF) [inline index.html:34]: The facet & the badge will display the captions instead

File: index.html [hunk 3]
```diff
@@ -38,19 +44,15 @@
     <div class="demo-banner">
       <span>This is a demo search page built with <u>Coveo</u></span>
     </div>
-    <atomic-search-interface
-      id="search"
-      analytics="false"
```
Comment (ThibodeauJF) [inline index.html:43]: it will start logging analytics

File: index.html [hunk 4]
```diff
@@ -38,19 +44,15 @@
...
-      pipeline="Website Search"
-    >
+    <atomic-search-interface id="search" pipeline="Website Search">
       <div class="logo">
         <img src="images/CoveoLogo.png" />
       </div>
       <atomic-search-box></atomic-search-box>
       <atomic-facet-manager>
-        <atomic-facet field="source" label="Content type"></atomic-facet>
+        <atomic-facet field="source" label="Content type" display-values-as="link"></atomic-facet>
```
Comment (ThibodeauJF) [inline index.html:53]: I think it's better, it's now a single select facet, like we have on our docs website

File: index.html [hunk 5]
```diff
@@ -85,6 +87,11 @@
             label="Long"
           ></atomic-numeric-range>
         </atomic-numeric-facet>
+        <atomic-facet
```
Comment (ThibodeauJF) [inline index.html:90]: Year is better than timeframe with our type of content

**Comment by salvarez07** [comment]
@ThibodeauJF Will there be a second PR for the UX and Atomic changes?

**Review by salvarez07 — APPROVED** [review]

**Pull request merged**